packages/monitor-v2/src/monitor-polymarket/index.ts

```diff
@@ -6,6 +6,7 @@ const logger = Logger;
 
 async function main() {
   const params = await initMonitoringParams(process.env);
+  const cache = { ancillaryData: new Map<string, string>() };
```

Comment:
Fetching the `ancillaryData` of each market takes time and can be reused between runs so I've decided to cache it

Reply:
Great call! I assume this only works in looping mode, right?

Reply:
Yes, only loop @mrice32